src/pages/registration/SupportModalContent.tsx

```diff
@@ -22,11 +24,83 @@ export const SupportModalContent = ({ registration }: SupportModalContentProps)
     onError: () => dispatch(setNotification({ message: t('feedback.error.get_tickets'), variant: 'error' })),
   });
 
+  const createSupportTicketMutation = useMutation(
+    async (message: string) => {
+      const newTicket = (await createTicket(registration.id, 'GeneralSupportCase', true)).data;
+      if (newTicket) {
+        await addTicketMessage(newTicket.id, message);
+      }
+    },
+    {
+      onSuccess: () => {
+        dispatch(setNotification({ message: t('feedback.success.send_message'), variant: 'success' }));
+        closeModal();
+      },
+      onError: () => dispatch(setNotification({ message: t('feedback.error.send_message'), variant: 'error' })),
+    }
+  );
+  const isLoading = ticketsQuery.isLoading || createSupportTicketMutation.isLoading;
+
+  const currentSupportTicket = ticketsQuery.data?.tickets
+    .filter((ticket) => ticket.type === 'GeneralSupportCase')
+    .at(-1);
+
   return (
-    <ActionPanelContent
-      tickets={ticketsQuery.data?.tickets ?? []}
-      refetchData={() => ticketsQuery.refetch()}
-      registration={registration}
-    />
+    <>
+      <Grid container spacing={3}>
+        <Grid item md={6} sm={12}>
+          <Typography variant={'h3'} marginBottom={2} gutterBottom>
+            {t('registration.support.self_help.header')}
+          </Typography>
+          <MuiLink
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              gap: '0.5rem',
+              mb: '1rem',
+              bgcolor: 'white',
+              padding: '0.5rem',
+              width: '100%',
+              borderRadius: '0.5rem',
+            }}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva'}>
+            <Typography>{t('footer.help_page')}</Typography>
+            <OpenInNew />
+          </MuiLink>
+          <Typography fontStyle={'italic'} marginBottom={2} gutterBottom>
+            {t('registration.support.self_help.description')}
+          </Typography>
+        </Grid>
+        <Grid item md={6} sm={12}></Grid>
```

Comment:
Tom tag?
```suggestion
```

Reply:
Det er en annen task for å fylle inn innhold der. Institusjonens egne hjelpesider som vi ikke har helt støtte for enda. Jeg sjekker om det blir bra visuelt uten å fylle ut hele grid'en.